packages/cardpay-sdk/sdk/safe-module.ts

```diff
@@ -222,26 +219,28 @@ export default abstract class SafeModule {
       multiSendTransaction.value,
       multiSendTransaction.data,
       multiSendTransaction.operation,
-      gasTokenAddress,
+      gasTokenAddress ?? AddressZero,
       true
     );
-    let gasCost = BigNumber.from(estimate.safeTxGas)
-      .add(BigNumber.from(estimate.baseGas))
-      .mul(BigNumber.from(estimate.gasPrice));
-
-    let token = new Contract(gasTokenAddress, ERC20ABI, this.ethersProvider);
-    let symbol = await token.symbol();
-    let balance = await token.callStatic.balanceOf(safeAddress);
-    let decimals = await token.callStatic.decimals();
-    if (balance.lt(gasCost)) {
-      throw new Error(
-        `Safe does not have enough balance to enable ${
-          this.name
-        } module. The gas token ${gasTokenAddress} balance of the safe ${safeAddress} is ${utils.formatUnits(
-          balance,
-          decimals
-        )}, the the gas cost is ${utils.formatUnits(gasCost, decimals)} ${symbol}`
-      );
+    if (gasTokenAddress) {
```

Comment:
Don't know why creating a safe was limited to gasToken only. We should be able to run it with eth